src/components/ModalSearch.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div style="height:60px">
+    <div
+      class="position-absolute bg-white py-3 px-3 px-md-4 width-full"
+      style="box-sizing: border-box;"
+    >
```

Comment:
theming related:
![searchbarcomp](https://user-images.githubusercontent.com/2432272/106742969-326a6500-6637-11eb-8760-29fb4bf245ec.PNG)

- the background of the search is not taking the theme in consideration
- once the background here changes it will reflect on the below button, so also we would need the button color to be updated accordingly in relation to the theme